cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -305,47 +392,39 @@ def prompt(
                 ),
             )
 
-        return cast(
-            List[TLMResponse],
-            self._event_loop.run_until_complete(
-                self._batch_prompt(prompt, capture_exceptions=False),
-            ),
+        return self._event_loop.run_until_complete(
+            self._batch_prompt(prompt, capture_exceptions=False),
         )
 
     def try_prompt(
         self,
         prompt: Sequence[str],
         /,
-    ) -> List[Optional[TLMResponse]]:
+    ) -> List[TLMResponse]:
         """
-        Gets response and trustworthiness score for any batch of prompts,
-        handling any failures (errors of timeouts) by returning None in place of the failures.
+        Gets response and trustworthiness score for any batch of prompts handling any failures (errors or timeouts).
 
-        The list returned will have the same length as the input list, if there are any
-        failures (errors or timeout) processing some inputs, the list will contain None in place of the response.
+        The list returned will have the same length as the input list. If there are any
+        failures (errors or timeouts) processing some inputs, the [TLMResponse](#class-tlmresponse) objects in the returned list will contain error messages and retryability information instead of the usual response.
 
-        This is the recommended way to get TLM responses and trustworthiness scores for big datasets of many prompts,
-        where some individual TLM responses within the dataset may fail. It ensures partial results are not lost.
+        This is the recommended approach for obtaining TLM responses and trustworthiness scores for large datasets with many prompts,
+        where some individual TLM responses within the dataset might fail. It ensures partial results are preserved.
 
         Args:
             prompt (Sequence[str]): list of multiple prompts for the TLM
         Returns:
-            List[Optional[TLMResponse]]: list of [TLMResponse](#class-tlmresponse) objects containing the response and trustworthiness score.
+            List[TLMResponse]: list of [TLMResponse](#class-tlmresponse) objects containing the response and trustworthiness score.
                 The returned list will always have the same length as the input list.
                 In case of TLM failure on any prompt (due to timeouts or other errors),
-                the return list will contain None in place of the TLM response for that failed prompt.
-                Use this to obtain TLM results for as many prompts as possible,
-                but you might miss out on certain error messages.
-                If you prefer to be notified immediately about any errors or timeouts when running many prompts,
+                the return list will include a [TLMResponse](#class-tlmresponse) with an error message and retryability information instead of the usual TLM response for that failed prompt.
```

Comment:
```suggestion
                the return list will include a [TLMResponse](#class-tlmresponse) with an error message and retryability information instead of the usual TLMResponse for that failed prompt.
```

Reply:
updated manually as the suggestion doesn't work anymore after the new commit